src/zepben/evolve/model/cim/iec61968/assetinfo/create_asset_info_components.py

```diff
@@ -0,0 +1,133 @@
+#  Copyright 2020 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from zepben.evolve import *
+
+
```

Comment:
each creator file will need an __all__ defined, for example:

`__all__ = ["create_cable_info", "create_no_load_test", ..., "create_transformer_tank_info"]`

Reply:
And then we'll need import statements in zepben.evolve.__init__.py - see my comment in conversations.